cypress/integration/delete-collection.ts

```diff
@@ -0,0 +1,24 @@
+describe("Let's delete from collection", () => {
+  it('Adds cards from search page', () => {
+    cy.visit('/')
+    cy.contains('Search').click()
+    cy.wait(2000)
+    const cardButtons = cy.get('figure > figcaption > div').find('>button')
+
+    cardButtons.each((cardButton) => {
+      cy.wrap(cardButton).click({ force: true })
+    })
+    cy.contains('My Collection').click()
+    cy.wait(2000)
```

Comment:
you never assert whether the cards actually got added to the collection? At the end here, you need to make sure all the cards you added in fact got added (use should())